swift/tools/tracing-config.lua

```diff
@@ -3,6 +3,53 @@ function RegisterExtractorPack(id)
     local relativeSwiftExtractor = extractorDirectory .. 'extractor'
     local swiftExtractor = AbsolutifyExtractorPath(id, relativeSwiftExtractor)
 
+    function indexOf(array, value)
+      for i, v in ipairs(array) do
+        if v == value then
+          return i
+        end
+      end
+      return nil
+    end
+
+    -- removes upsupported CLI arg including the following how_many args
+    function strip_unsupported_arg(args, arg, how_many)
+      index = indexOf(args, arg)
```

Comment:
```suggestion
      local index = indexOf(args, arg)
```
otherwise you're introducing a global variable in your scope (yeah, I know, that part is not great about Lua)